pkgs/by-name/ca/cambalache/package.nix

```diff
@@ -60,18 +72,18 @@ python3.pkgs.buildPythonApplication rec {
     # For extra widgets support.
```

Comment:
Looks like 3rd party libraries are now supported. We should look into what that requires and how it interacts with Nix's lack of FHS – what paths and environment files does it use to finding the catalogs?